deprish_to_eia.py

```diff
@@ -164,34 +172,59 @@ def get_fuzzy_matches(deprish_df, mul_df, key_deprish, key_mul, threshold=75):
 
 def match_merge(deprish_df, mul_df, key_deprish, key_mul):
     """Generate fuzzy matches and merge relevant colums from eia."""
-    deprish_cols = [
-        'utility_id_ferc1', 'utility_id_pudl', 'utility_name_ferc1', 'state',
-        'plant_id_pudl', 'plant_name', 'report_year', 'plant_name_match']
-    mul_cols = [
-        'record_id_eia', 'plant_name_new', 'plant_name_eia', 'plant_id_eia',
-        'report_year', 'plant_part', 'generator_id', 'unit_id_pudl',
-        'prime_mover_code', 'energy_source_code_1',
-        'technology_description', 'ferc_acct_name', 'utility_id_eia',
-        'true_gran', 'appro_part_label', 'appro_record_id_eia',
-    ]
     logger.info("Merging fuzzy matches.")
     match_merge = pd.merge(
         get_fuzzy_matches(
             deprish_df, mul_df,
             key_deprish=key_deprish, key_mul=key_mul,
-            threshold=75)[deprish_cols],
+            threshold=75)[DEPRISH_COLS + ['plant_name_match']],
         mul_df.reset_index().drop_duplicates(
-            subset=['report_year', 'plant_name_new'])[mul_cols],
+            subset=['report_year', 'plant_name_new'])[MUL_COLS],
         left_on=['report_year', 'plant_name_match'],
         right_on=['report_year', key_mul], how='left')
     logger.info(f"Matching resulted in {len(match_merge)} connections.")
     return match_merge
 
 
-def match_deprish_eia(file_path_mul, file_path_deprish, sheet_name_deprish):
+def add_overrides(deprish_match, file_path_deprish, sheet_name_output):
+    """
+    Add the overrides into the matched depreciation records.
+
+    Args:
+        deprish_match (pandas.DataFrame):
+        file_path_deprish (os.PathLike): path to the excel workbook which
+           contains depreciation data.
+       sheet_name_output (string): name of the excel tab which the matches
+          will be output.
+    Return:
```

Comment:
A blank line is required between the `Args:` and `Returns:` sections for docstrings to parse correctly in documentation generation. Also I think it needs to be Return**s**: and the (string) needs to be (str) if it's going to be parsed right. I suspect these docstrings won't end up getting fed into Sphinx any time soon, but it would be nice if we could get in the habit of writing parseable docstrings so there isn't a bunch of tedious cleanup to do when we're trying to make a release with readable documentation.